aptos-move/framework/aptos-stdlib/sources/data_structures/smart_table.move

```diff
@@ -288,8 +319,7 @@ module aptos_std::smart_table {
         let index = hash % (1 << (level + 1));
         if (index < num_buckets) {
             // in existing bucket
-            index
-        } else {
+            index } else {
```

Comment:
Is this a bug?

Reply:
This should be a bug introduced during the optimization of complex branch statements in this iteration.